src/cf-base.less

```diff
@@ -297,7 +315,6 @@ dl,
 table,
 figure {
     margin-top: 0;
-    // Spacing has been eyeballed to be 30px from the baseline.
     margin-bottom: unit(20px / @base-font-size-px, em);
```

Comment:
I don't think so because 20px matches the main headings and when we were discussing it I remember @nataliafitzgerald specifically asking me if it was 20px.